doc/sphinx/methods.rst

```diff
@@ -361,6 +361,49 @@ To calculate Bader charges from a cube file, a ``Volume`` object should be prepa
     m.calculate()
 
 
+.. index::
+    single: methods; DDEC6
+    
+DDEC6
+----------------
+
+DDEC6 charges are stockholder-like charge partitioning method introduced in `[Manz2016] <https://doi.org/10.1039/C6RA04656H>`_. Proatom densities should be provided for DDEC6 method. Proatom densities generated using `horton`_ can be passed on as an argument for the constructor of the DDEC6 object. For DDEC6 calculations, it is recommended that electron densities are imported from cube files. Because a lot of numerical integrations are present in DDEC6 algorithm, fine grid is necessary to obtain satisfying results.
```

Comment:
> "For DDEC6 calculations, it is recommended that electron densities are imported from cube files. Because a lot of numerical integrations are present in DDEC6 algorithm, fine grid is necessary to obtain satisfying results."

The DDEC6 algorithm requires many numerical integrations so a fine grid is necessary to obtain accurate results. Calculating the electron density on a fine grid using a `Volume` object is slow, therefore we recommend that electron densities are imported from cube files.

Because a lot of numerical integrations are present in DDEC6 algorithm, fine grid is necessary to obtain satisfying results.

Reply:
I've updated this. Thanks for the comment!